src/test/cls_lock/test_cls_lock.cc

```diff
@@ -502,23 +502,25 @@ TEST(ClsLock, TestExclusiveEphemeralBasic) {
   ASSERT_EQ(0, l1.lock_exclusive_ephemeral(&ioctx, oid1));
   ASSERT_EQ(0, ioctx.stat(oid1, &size, &mod_time));
   sleep(2);
+  bool r1_expired = lock_expired(&ioctx, oid1, lock_name1);
   int r1 = l1.unlock(&ioctx, oid1);
-  EXPECT_TRUE(r1 == 0 || ((r1 == -ENOENT) && (lock_expired(&ioctx, oid1, lock_name1))))
-    << "unlock should return 0 or -ENOENT return: " << r1;
+  EXPECT_TRUE( r1 == 0 )
+    << "unlock should return 0 return: " << r1 << " expired: " << r1_expired;
```

Comment:
What is the purpose of printing `r1_expired` and, going further, calling `lock_expired` at all?  You aren't asserting on `r1_expired` so it's essentially dead code because no one is looking at the output if the tests pass.

Reply:
Done